unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/NFTIcons/NftPageView.cs

```diff
@@ -29,9 +36,38 @@ public void SetPageElementsContent(NFTIconComponentModel[] nftModels, string[] i
         }
     }
 
+    public void ConfigureNFTItemInfo(NFTItemInfo nftItemInfoModal, WearableItem[] wearableItems, bool showCategoryInfo)
+    {
+        for (int i = 0; i < nftElements.Length; i++)
+        {
+            if (wearableItems[i] != null)
+                nftElements[i].ConfigureNFTItemInfo(nftItemInfoModal, wearableItems[i], showCategoryInfo);
+        }
+    }
+
+    public void CloseAllNFTItemInfos()
+    {
+        for (int i = 0; i < nftElements.Length; i++)
+            nftElements[i].SetNFTItemInfoActive(false);
+    }
+
     private void ClickOnBuyWearable(int index)
     {
-        OnClickBuyNft?.Invoke(nftIds[index]);
+        OnClickBuyNft?.Invoke(nftIds[index].Item1, nftIds[index].Item2);
+    }
+
+    private void ClickOnDetailInfo(int index, bool showInLeftSide)
+    {
+        if (IsNftInfoActived)
+            nftElements[index].SetNFTItemInfoActive(true, showInLeftSide);
+    }
+
+    private void FocusNftItem(bool isFocused)
+    {
+        if (!isFocused)
```

Comment:
inverting the `if` would make it more readable

Reply:
The fact is that it's an if for the return statement, there might be new code added below so I'd rather stay with this if :)

Reply:
ok, you can keep it. But I would rather go with the YAGNI meta-principle approach and change it to `if (!isFocused) return;` when needed